security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,15 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-0-0"]
+== OpenShift File Integrity Operator 1.0.0
+
+The following advisory is available for the OpenShift File Integrity Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:0037[RHBA-2023:0037 OpenShift File Integrity Operator Bug Fix Update]
+
+The File Integrity Operator is now stable and the release channel is upgraded to `release-1.0`. Future releases will follow link:https://semver.org/[Semantic Versioning]. To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-configuring.html#file-integrity-operator-release-channel_file-integrity-operator[Configuring the File Integrity Operator release channel].
```

Comment:
Should be:
The File Integrity Operator is now stable and the release channel is upgraded to `v1`